pexec/managed_process.go

```diff
@@ -40,17 +40,46 @@ type ManagedProcess interface {
 // NewManagedProcess returns a new, unstarted, from the given configuration.
 func NewManagedProcess(config ProcessConfig, logger golog.Logger) ManagedProcess {
 	logger = logger.Named(fmt.Sprintf("process.%s_%s", config.ID, config.Name))
+
+	var stopSig syscall.Signal
```

Comment:
[nit] drop this into validate and add tests

Reply:
Do you mean MOVE this to the config.Validate() method in process.go? Or just ALSO add tests/checks over there? I don't think it can be moved, as I'm parsing into an entirely different type than the string value from the config.